tests/test_misc.py

```diff
@@ -254,7 +254,7 @@ def mock_getblock(r):
 
 
 def test_ping(node_factory):
-    l1, l2 = node_factory.line_graph(2, fundchannel=False)
+    l1, l2 = node_factory.line_graph(2)
 
     def ping_tests(l1, l2):
         # 0-byte pong gives just type + length field.
```

Comment:
```suggestion
        # 0-byte pong gives just type + length field.
        
        l1.rpc.ping(l2.info['id'])
```

Maybe we can test also if the node responds correctly to the ping that should be run without any problem?